torchao/quantization/quant_api.py

```diff
@@ -1474,13 +1484,16 @@ class Float8WeightOnlyConfig(AOBaseConfig):
 
     Args:
         weight_dtype (torch.dtype): The target data type for weight quantization. Default is torch.float8_e4m3fn.
+        packing_format (PackingFormat): The packing_format for the quantized float8 data. Currently
```

Comment:
should we hide this from the config until there is more than one option, and at that point add the plain one as default?